docs/source/oci/examples/index.rst

```diff
@@ -0,0 +1,46 @@
+.. _ociexamples:
+
+Example Policies
```

Comment:
The amount of example policies and the way you've organized them is delightful :heart:

Reply:
Thanks!